client/my-sites/plan-features/actions.jsx

```diff
@@ -50,6 +53,10 @@ const PlanFeaturesActions = ( {
 		if ( isLandingPage ) {
 			buttonText = translate( 'Select', { context: 'button' } );
 		}
+		if ( isInSignup && ( abtest( 'signupPlansCallToAction' ) === 'modified' ) ) {
+			buttonText = 'Start with ' + planName;
```

Comment:
Apparently you can't add variables to translations. I'm going to ship it as is. Any thoughts about how you could handle this?